packages/fluentui/react-northstar/src/components/Menu/MenuItem.tsx

```diff
@@ -417,8 +402,8 @@ export const MenuItem = compose<'a', MenuItemProps, MenuItemStylesProps, {}, {}>
       // which cause a broken behavior: for e.g. when it is needed to focus submenu trigger on ESC.
       // TODO: all DOM post-effects should be applied at componentDidMount & componentDidUpdated stages.
       onStateChanged && onStateChanged();
-      _.invoke(props, 'onMenuOpenChange', e, {
-        ...props,
+      _.invoke(allProps, 'onMenuOpenChange', e, {
```

Comment:
This looks really scary :(

Reply:
Yep :( I feel like it's better to just say
```
props = {...parentProps, ...props} 
```